src/Scripting/Core/Scripting.csproj

```diff
@@ -47,6 +47,9 @@
     <Compile Include="..\..\Compilers\Core\Portable\FileSystem\RelativePathResolver.cs">
       <Link>Resolvers\RelativePathResolver.cs</Link>
     </Compile>
+    <Compile Include="..\..\Compilers\Shared\CoreClrShim.cs">
+      <Link>CoreClrShim.cs</Link>
```

Comment:
Is CoreClrShim still used elsewhere? If not, let's just move it here instead of linking it.